Python/sysmodule.c

```diff
@@ -374,6 +374,23 @@ sys_exc_info_impl(PyObject *module)
 }
 
 
+/*[clinic input]
+sys.unraisablehook
+
+    args: object
+    /
+
+Print an unraisable exception.
```

Comment:
Can you explain what `args` should be here? (Apparently people read these docs :) )

Reply:
Sure, done.